functional-samples/ai.gemini-on-device-summarization/sidepanel/index.js

```diff
@@ -1,3 +1,6 @@
+import DOMPurify from '../node_modules/dompurify/dist/purify.es.mjs';
+import { marked } from '../node_modules/marked/marked.min';
```

Comment:
It's quite unusual to import from a relative path like this. Could you look at what setup would be needed so you can just import from "marked"?

Reply:
fixed